lib/json/json.go

```diff
@@ -54,15 +54,19 @@ import (
 // (e.g. it implements both Iterable and HasFields), the first case takes precedence.
 // Encoding any other value yields an error.
 //
-// def decode(x):
+// def decode(x[, default]):
 //
-// The decode function accepts one positional parameter, a JSON string.
+// The decode function has one required positional parameter, a JSON string.
 // It returns the Starlark value that the string denotes.
 // - Numbers are parsed as int or float, depending on whether they
 //   contain a decimal point.
 // - JSON objects are parsed as new unfrozen Starlark dicts.
 // - JSON arrays are parsed as new unfrozen Starlark lists.
-// Decoding fails if x is not a valid JSON string.
+// If `x` is not a valid JSON string and the optional, positional-only `default`
```

Comment:
More concisely:
```
// If x is not a valid JSON string the behavior depends
// on the "default" parameter: if present, Decode
// returns its value; otherwise Decode fails.
```

Don't use backquotes in Go doc comments. If the result would be ambiguous (as with "the default parameter") then use double quotes around the name.